src/modules/UI/components/FlipInput/ExchangedFlipInput2.js

```diff
@@ -147,6 +147,13 @@ export class ExchangedFlipInput extends Component<Props, State> {
     this.props.onExchangeAmountChanged({ exchangeAmount, nativeAmount })
   }
 
+  isFiatOnTop = () => {
+    if (this.props.isFiatOnTop) {
+      return this.props.isFiatOnTop
+    }
+    return this.state.exchangeSecondaryToPrimaryRatio !== false
```

Comment:
exchangeSecondaryToPrimaryRatio is a string. This comparison checks it as a boolean. You should be checking it to see if it is equal to 0 using a line such as 

`bns.eq(this.state.exchangeSecondaryToPrimaryRatio, '0')`

Reply:
Updated